encode_datasets.py

```diff
@@ -1282,9 +1280,10 @@ def csqa2_process_test(file, dataset, kind):
         if '?' not in question:
             question = question + "?"
         id=questions[row][1]
+        answer=["-"]
 
         fmeta.write(f"{id} \n")
```

Comment:
@yeganehkordi could you update the meta files for CSQA2.0 such that the 2nd column contains the index of correct answers? (represented as "A", "B", ..).

Reply:
Sure! Will do.